README.md

```diff
@@ -139,22 +142,22 @@ space_base_demo_to_plot = {
 
 ## Analysis of the hyperparameters
 
- Here is an excerpt:
+Here is an excerpt:
 
 <p align="center">
   <img src="hyperparameters_scatter_matrix.png">
 </p>
 
 This could help to redefine the hyperparameters and to narrow them down successively, relaunching the meta-optimization on refined spaces.
 
-
 ## Best result
 
-The final accuracy is of 67.61% in average on the 100 fine labels, and is of 77.31% in average on the 20 coarse labels.
-The results are comparable to the ones in the middle of [that list](http://rodrigob.github.io/are_we_there_yet/build/classification_datasets_results.html#43494641522d313030), under the CIFAR-100 section. 
+The final accuracy is 67.61% on average for the 100 fine labels, and is 77.31% on average for the 20 coarse labels.
+These results are comparable to the ones in the middle of [this list](http://rodrigob.github.io/are_we_there_yet/build/classification_datasets_results.html#43494641522d313030), under the CIFAR-100 section.
```

Comment:
what is the significance of this list? might be worth contextualizing. maybe it's due to my lack of familiarity, but this felt like an arbitrary list on the internet to compare against :)